pcsx2/gui/GlobalCommands.cpp

```diff
@@ -184,15 +184,15 @@ namespace Implementations
 			default: break;
 		}
 
-		Console.WriteLn(L"(GSwindow) Aspect ratio: %s.", WX_STR(arts));
+		OSDlog( Color_StrongBlue, true, "(GSwindow) Aspect ratio: %s.", WX_STR(wxString(arts)));
```

Comment:
Sorry, I meant that as gotcha, I understand.  Not as double check your work.  I made the change, thank you.